server/src/main/java/io/druid/server/emitter/ParametrizedUriEmitterModule.java

```diff
@@ -63,6 +64,7 @@ public Emitter getEmitter(
     if (sslContext != null) {
       builder.setSslContext(new JdkSslContext(sslContext, true, ClientAuth.NONE));
     }
+    builder.setThreadFactory(new DefaultThreadFactory("parametrized-emitter-http-client", true));
```

Comment:
`io.druid.java.util.common.concurrent.Execs#makeThreadFactory(java.lang.String)` should be used